quickwit/quickwit-search/src/root.rs

```diff
@@ -353,7 +353,7 @@ fn validate_sort_by_field(
     has_timestamp_format: bool,
     schema: &Schema,
 ) -> crate::Result<()> {
-    if field_name == "_score" {
+    if ["_score", "_shard_doc", "_doc"].contains(&field_name) {
```

Comment:
oh, thank you for fixing this; I totally forgot it.

Do we have tests on this?

Reply:
I don't think so